pkg/server/proto/server.proto

```diff
@@ -3165,6 +3194,12 @@ message Deployment {
   // Resources that this deployment has created or manages.
   repeated DeclaredResource declared_resources = 19;
 
+  // Variable refs store the final value used on the operation for each variable
+  // defined in the waypoint.hcl. Any variables with `sensitive` set in the
+  // waypoint.hcl will have a value encoded as SHA256 so the user can verify
+  // the value used.
+  map<string, Variable.Ref> variable_refs = 21;
```

Comment:
I don't think we need these on operations, because the operation contains a reference to the job (`job_id`), and you can get the variables from the job. For APIs, we can eagerly do that and put it on the response perhaps. But I think no reason to have it here?